### PR TITLE
[SPARK-28869][DOCS][FOLLOWUP] Add direct relationship between configs for rolling event log

### PR DESCRIPTION
**Pull request by HeartSaVioR**
### What changes were proposed in this pull request?

This patch addresses the post-hoc review comment linked here - https://github.com/apache/spark/pull/25670#discussion_r373304076

### Why are the changes needed?

We would like to explicitly document the direct relationship before we finish up structuring of configurations.

### Does this PR introduce any user-facing change?

No.

### How was this patch tested?

N/A

### CODE REVIEW

**Comment by HeartSaVioR** [comment]
cc. @HyukjinKwon

**Comment by HeartSaVioR** [comment]
retest this, please

**Comment by HyukjinKwon** [comment]
retest this please

**Review by HyukjinKwon — APPROVED** [review]
Thanks @HeartSaVioR.

File: core/src/main/scala/org/apache/spark/internal/config/package.scala [hunk 1]
```diff
@@ -198,7 +198,8 @@ package object config {
 
   private[spark] val EVENT_LOG_ROLLING_MAX_FILE_SIZE =
     ConfigBuilder("spark.eventLog.rolling.maxFileSize")
-      .doc("The max size of event log file to be rolled over.")
+      .doc(s"Precondition: ${EVENT_LOG_ENABLE_ROLLING.key}. " +
```
Comment (HyukjinKwon) [inline core/src/main/scala/org/apache/spark/internal/config/package.scala]: Okay, I saw the new PR https://github.com/apache/spark/pull/27575. We could match how we describe there. Shall we also fix `docs/configuration.md`?

Comment (HeartSaVioR) [reply core/src/main/scala/org/apache/spark/internal/config/package.scala]: Nice catch! I missed we have equivalent description in doc as well. Will update as well.

**Comment by HeartSaVioR** [comment]
Picked existing style from #27575, and updated here as well.

**Comment by HyukjinKwon** [comment]
Thanks @HeartSaVioR for folliwing up this one.

**Comment by HyukjinKwon** [comment]
Merged to master and branch-3.0.

**Comment by HeartSaVioR** [comment]
Thanks for reviewing and merging!

**Pull request closed without merging**